media/stump-radio/package.lisp

```diff
@@ -0,0 +1,9 @@
+(in-package :common-lisp-user)
+
+(defpackage #:stump-radio
+  (:use #:cl #:stumpwm)
```

Comment:
Really should avoid `:use` for StumpWM, instead `:import-from :stumpwm :symbols :you :need`
```suggestion
  (:use #:cl)
  (:import-from #:stumpwm #:message #:defcommand)
```